dev/src/TestUserInput.ts

```diff
@@ -32,10 +32,11 @@ export class TestUserInput implements types.TestUserInput {
         return this._onDidFinishPromptEmitter.event;
     }
 
-    public async runWithInputs(inputs: (string | RegExp | types.TestInput)[], callback: () => Promise<void>): Promise<void> {
+    public async runWithInputs<T>(inputs: (string | RegExp | types.TestInput)[], callback: () => Promise<T>): Promise<T> {
```

Comment:
Gotta update here, too: https://github.com/microsoft/vscode-azuretools/blob/master/dev/index.d.ts#L159

Reply:
Ah! I saw src/index.ts and it doesn't need updating, thought index.d.ts was generated. I'll add it.